app/src/main/java/com/keylesspalace/tusky/components/scheduled/ScheduledTootAdapter.kt

```diff
@@ -31,9 +33,18 @@ interface ScheduledTootAction {
 
 class ScheduledTootAdapter(
         val listener: ScheduledTootAction
-) : RecyclerView.Adapter<ScheduledTootAdapter.TootViewHolder>() {
+) : PagedListAdapter<ScheduledStatus, ScheduledTootAdapter.TootViewHolder>(
+        object: DiffUtil.ItemCallback<ScheduledStatus>(){
```

Comment:
It's really non-obvious that it's an argument to the constructor but idk if it justifies a separate variable.

Reply:
Also would be cool to have a default implementation which works for all items with IDs

Reply:
yes, next time I need it I will extract it